### PR TITLE
Add resource translation readme file.

### PR DESCRIPTION
**Pull request by rghetia**

### CODE REVIEW

File: RESOURCE.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# RESOURCES
+
+Stackdriver has defined [resource type](https://cloud.google.com/monitoring/api/resources) for monitoring and for each resource type there
```
Comment (songy23) [inline RESOURCE.md]:
```suggestion
Stackdriver has defined [resource types](https://cloud.google.com/monitoring/api/resources) for monitoring and for each resource type there
```

**Comment by rghetia** [comment]
Ideally this should be in language independent repo for stackdriver. Since we don't have one. I have created this in this repo.

File: RESOURCE.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+are mandatory resource labels. OpenCensus has defined [standard resource](https://github.com/census-instrumentation/opencensus-specs/blob/master/resource/StandardResources.md)
+types and labels. 
+This document describes the translation from OpenCensus resources to Stackdriver resources
+performed by this exporter.
+
+## Mapping between Stackdriver and OpenCensus Resources
+
+### k8s_container
```
Comment (songy23) [inline RESOURCE.md]: You can also mention `k8s_container` takes precedence, i.e GKE will be mapped to `k8s_container` and won't have GCP specific labels.

File: RESOURCE.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+|                     | k8s.cluster.name   | cluster_name   |
+|                     | k8s.namespace.name | namespace_name |
+|                     | k8s.pod.name       | pod_name       |
+|                     | container.name     | container_name |
+
+
+### gcp_instance
+**condition:** cloud.provider == gcp
+
+| Item                | OpenCensus         | Stackdrive     |
```
Comment (dinooliva) [inline RESOURCE.md]: Stackdrive  -> Stackdriver

File: RESOURCE.md (hunk 1 above)
Comment (rghetia) [reply RESOURCE.md]: done.

File: RESOURCE.md (hunk 2 above)
Comment (rghetia) [reply RESOURCE.md]: done.

File: RESOURCE.md (hunk 3 above)
Comment (rghetia) [reply RESOURCE.md]: done.

**Pull request merged**